cmd/soroban-cli/src/commands/tx/fetch/fee.rs

```diff
@@ -0,0 +1,177 @@
+        let tx_meta = resp.result_meta.clone().unwrap();
+
+        let fee = tx_result.fee_charged;
+        let (non_refundable_resource_fee, refundable_resource_fee) = match tx_meta.clone() {
+            TransactionMeta::V0(_) | TransactionMeta::V1(_) | TransactionMeta::V2(_) => {
+                (DEFAULT_FEE_VALUE, DEFAULT_FEE_VALUE)
+            }
+            TransactionMeta::V3(meta) => {
+                if let Some(soroban_meta) = meta.soroban_meta {
+                    match soroban_meta.ext {
+                        SorobanTransactionMetaExt::V0 => (DEFAULT_FEE_VALUE, DEFAULT_FEE_VALUE),
+                        SorobanTransactionMetaExt::V1(v1) => (
+                            v1.total_non_refundable_resource_fee_charged,
+                            v1.total_refundable_resource_fee_charged,
+                        ),
+                    }
+                } else {
+                    (DEFAULT_FEE_VALUE, DEFAULT_FEE_VALUE)
+                }
+            }
+        };
+
+        let fee_table = FeeTable::new(fee, non_refundable_resource_fee, refundable_resource_fee);
+
+        match self.output {
+            FeeOutputFormat::Json => {
+                println!("{}", serde_json::to_string(&fee_table)?);
+            }
+            FeeOutputFormat::JsonFormatted => {
+                println!("{}", serde_json::to_string_pretty(&fee_table)?);
+            }
+            FeeOutputFormat::Table => {
+                fee_table.print();
+            }
+        }
+
+        Ok(())
+    }
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct FeeTable {
+    pub fee: i64,
+    pub resource_fee: i64,
+    pub inclusion_fee: i64,
+    pub non_refundable_resource_fee: i64,
+    pub refundable_resource_fee: i64,
+}
+
+impl FeeTable {
+    fn new(fee: i64, non_refundable_resource_fee: i64, refundable_resource_fee: i64) -> Self {
+        let resource_fee = non_refundable_resource_fee + refundable_resource_fee;
+        FeeTable {
+            fee,
+            resource_fee,
+            inclusion_fee: fee - resource_fee,
+            non_refundable_resource_fee,
+            refundable_resource_fee,
+        }
+    }
+
+    fn print(&self) {
+        let table_format = FormatBuilder::new()
+            .column_separator('│')
+            .borders('│')
+            .separators(&[LinePosition::Top], LineSeparator::new('─', '─', '┌', '┐'))
+            .separators(
+                &[LinePosition::Intern],
+                LineSeparator::new('─', '─', '├', '┤'),
+            )
+            .separators(
+                &[LinePosition::Bottom],
+                LineSeparator::new('─', '─', '└', '┘'),
+            )
+            .padding(1, 1)
+            .build();
+
+        let mut table = Table::new();
+
+        table.set_format(table_format);
+
+        table.add_row(Row::new(vec![Cell::new(&format!("tx.fee: {}", self.fee))
+            .style_spec("b")
+            .with_hspan(3)]));
+
+        if self.resource_fee == 0 {
```

Comment:
Added this piece to simplify the fee table for classic txns. Does this look ok, or do we want to add some additional info? _(this is for a manage data tx)_

```
┌─────────────────┐
│ tx.fee: 100     │
└─────────────────┘
```

Reply:
Hi @elizabethengelman, 

For classic, that looks good. 

Also I think we should show:
- Max fee: Max fee specified in the transaction itself – the maximum XLM amount the source account willing to pay. Each transaction sets a fee that is paid by the source account. 
- Transaction fee (what you have): Transaction fee is what is actually charged during the transaction execution, which can be lower than the Max fee specified. (Transaction fee  = Resource fee + Inclusion fee)